pkg/provisioning/service.go

```diff
@@ -418,6 +461,76 @@ func (s *Service) createProcessors(ctx context.Context, r *rollback.R, parentID
 	return nil
 }
 
+func (s *Service) deleteOldPipelines(ctx context.Context, pipelines []string) {
+	var err error
+	plMp := s.pipelineService.List(ctx)
+	for id, pl := range plMp {
+		if !slices.Contains(pipelines, id) && pl.ProvisionedBy == pipeline.ProvisionTypeConfig {
+			err = s.deletePipelineWithoutRollback(ctx, pl)
+			if err != nil {
+				s.logger.Warn(ctx).Str("pipelineID", id).Msg("could not delete pipeline")
```

Comment:
We have constants for common log fields. I would also add more information to this error to let the user know what's going on. Along the way of "failed to delete pipeline provisioned by a config file, the pipeline is probably in a broken state, Conduit will try to remove it again next time it runs"
```suggestion
				s.logger.Warn(ctx).Str(log.PipelineIDField, id).Msg("could not delete pipeline")
```